docs/usage.md

```diff
@@ -187,16 +197,55 @@ params {
 
 ### `--fasta`
 
-If you prefer, you can specify the full path to your reference genome when you run the pipeline:
+Full path to fasta file containing reference genome (*mandatory* if `--genome` is not specified). If you don't have a BWA index available this will be generated for you automatically. Combine with `--save_reference` to save BWA index for future runs.
+
+```bash
+--fasta '[path to FASTA reference]'
+```
+
+### `--bwa_index`
```

Comment:
isn't it `--bwa`?

Reply:
This needs to come out completely because we are using `bowtie2`. Still need to update the usage docs properly but wanted to get the primary parameters and tests working first.

Reply:
Fixed https://github.com/nf-core/viralrecon/pull/11/commits/741e9e60367b05b11d2fc448ef67e9687129ae00